src/makeRobotAccountant.js

```diff
@@ -6,7 +6,19 @@
  */
 
 function makeRobotAccountant() {
-  // write code here
+  let counter = 0;
+
+  const getSum = a => {
```

Comment:
It's more convenient to always put function params in brackets
```suggestion
  const getSum = (a) => {
```